### ISSUE TITLE
MRCM Range search - wrong language

### ISSUE THREAD

**Issue by liquid36** [IHTSDO/snowstorm#589]
Hi! me again...
I trying the MRCM endpoints. When i want to search for attributes values in Spanish, Snowstorm insist to search in English. Example: 

```
curl -X 'GET' \
  'http://localhost:8080/mrcm/MAIN%2FSNOMEDCT-ES%2FSNOMEDCT-AR/attribute-values/263502005?contentType=NEW_PRECOORDINATED&termPrefix=crónico' \
  -H 'accept: application/json' \
  -H 'Accept-Language: es'
```

But if i change _crónico_ by _chronic_, i get values.

Searching for query logs in Elasticsearch, i found that Snowstorm is setting the languageCode to **en**:

```
{
    "from": 0,
    "size": 10000,
    "query": {
        "bool": {
            "must": [
                ... BRANCH CRITERIA 
                {
                    "terms": {
                        "languageCode": [
                            "en"
                        ],
                        "boost": 1.0
                    }
                },
                {
                    "term": {
                        "active": {
                            "value": true
                        }
                    }
                }
            ],
            "filter": [
                {
                    "bool": {
                        "must": [
                            {
                                "bool": {
                                    "should": [
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        },
                                        {
                                            "bool": {
                                                "filter": [
                                                    {
                                                        "query_string": {
                                                            "query": "chronic*",
                                                            "fields": [
                                                                "termFolded^1.0"
                                                            ],
                                                            "default_operator": "and",
                                                            "boost": 2.0
                                                        }
                                                    }
                                                ],
                                                "boost": 1.0
                                            }
                                        }
                                    ],
                                    "boost": 1.0
                                }
                            }
                        ],
                        "boost": 1.0
                    }
                }
            ],
            "boost": 1.0
        }
    },
    "version": true,
    "_source": {
        "includes": [
            "descriptionId",
            "conceptId"
        ],
        "excludes": []
    },
    "sort": [
        {
            "termLen": {
                "order": "asc",
                "mode": "min"
            }
        },
        {
            "_score": {
                "order": "asc"
            }
        }
    ]
}
```
It appears that the **Accept-Language header** is only utilized for the response, rather than the query.

**Comment by kaicode** [comment]
@liquid36 I was able to reproduce this problem. Yes it looks like Snowstorm currently limits the MRCM attribute search stage to English terms. I've created a code fix for this on branch `mrcm-language-fix`. Would you able to build and test this?

**Comment by liquid36** [comment]
Could you provide me instruction about how to build it? I'm not an expert on maven. 
A Dockerfile would be great.

**Comment by liquid36** [comment]
It's working as i expected! Thanks very much. You could release the fix.

**Comment by kaicode** [comment]
Great! Thanks for testing. I will raise a ticket for the technical team to merge this in.

**Comment by kaicode** [comment]
Raised internal ticket INFRA-12853

**Comment by liquid36** [comment]
Was this fix included in the last release?

**Comment by kaicode** [comment]
This fix did not make it into the last release. It's still in the develop branch at the moment.

**Comment by kaicode** [comment]
The fix for this issue is included in release 10.4.2.
